src/case_runners/case_runner.jl

```diff
@@ -13,14 +13,14 @@ end
 @doc raw"""Run the GenX in the given folder
 case - folder for the case
 """
-function run_genx_case!(case::AbstractString)
+function run_genx_case!(case::AbstractString, optimizer::Any=HiGHS.Optimizer)
```

Comment:
`::Any` here is correct, because users can pass one of the following:

 * a `Type{<:MOI.AbstractOptimizer}`, like `HiGHS.Optimizer`
 * a function which when called with zero arguments returns an optimizer, like `() -> Gurobi.Optimizer(env)`